README.aspnetcore-nightly.md

```diff
@@ -12,14 +12,14 @@ The [.NET Core Docker samples](https://github.com/dotnet/dotnet-docker/blob/mast
 
 # Linux amd64 tags
 
-- [`1.0.13-jessie`, `1.0.13`, `1.0` (*1.0/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.0/jessie/runtime/Dockerfile)
```

Comment:
Is nightly applicable anymore?  Should these artifacts and corresponding Docker Hub repo be cleaned up?  I'm not suggesting doing this with the PR, rather a followup.

Reply:
We don't build the nightly repo anymore. Do you know if there is a way to completely deprecate a Docker hub repo?

Reply:
We can update the readme and possibly do delete it.  I would suggest we update the description to indicate it is deprecated and put a timeline on when it will be removed.